vpr/src/base/netlist.h

```diff
@@ -663,6 +663,17 @@ class Netlist {
     //Returns a range consisting of all pins in the netlist
     pin_range pins() const;
 
+    
+    /*
+     * id Checks
+     */
```

Comment:
Should probably add a comment giving a little more context about what these do (i.e. Validates that the specified ID is valid in the current netlist state)